.pre-commit-config.yaml

```
@@ -0,0 +1,8 @@
+repos:
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  # Ruff version.
+  rev: v0.12.5
+  hooks:
+    - id: ruff-format
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)